alerting/src/main/kotlin/org/opensearch/alerting/MonitorRunnerService.kt

```diff
@@ -218,32 +224,31 @@ object MonitorRunnerService : JobRunner, CoroutineScope, AbstractLifecycleCompon
     }
 
     override fun runJob(job: ScheduledJob, periodStart: Instant, periodEnd: Instant) {
```

Comment:
Can we add comments here on why we can only have 1 job runner and not a separate one for workflows? And we can talk about if the job scheduler plugin takes care of this, we can go back to having 2 job runners.